docs/install.md

```diff
@@ -12,7 +12,7 @@ Before you start:
 
 ---
 
-Download and install the latest Magisk app. We use the app to gather some information about your device. In the home screen, you should see this:
+Download the [latest Magisk APK](https://github.com/topjohnwu/Magisk/releases) and install it on your device using `adb`. We use the app to gather some information about your device. In the home screen, you should see this:
```

Comment:
`adb` isn't really needed to install the apk, you can use the device's installer

Reply:
Fair point. Updated.

Reply:
> Fair point. Updated.

Nice :D